velox/common/memory/MemoryPool.h

```diff
@@ -467,7 +467,7 @@ class MemoryPool : public std::enable_shared_from_this<MemoryPool> {
 
   /// Invoked to generate a descriptive memory usage summary of the entire tree.
   /// MemoryPoolImpl::treeMemoryUsage()
-  virtual std::string treeMemoryUsage() const = 0;
+  virtual std::string treeMemoryUsage(bool detail = false) const = 0;
```

Comment:
I feel detail is not descriptive enough as we only use this flag to print the whole tree regardless of its emptiness. Should we call it skipEmptyPool or skipEmpty and default to true or something similar?

Reply:
Yeah it is better.